### PR TITLE
Add 1 Icon

### PR DESCRIPTION
**Pull request by Hamster45105**
# Description

1 icon has been added.

### Added
ownCloud (`com.owncloud.android`)

## Note
Credit for the ownCloud icon goes to @eltacogatito who originally tried to add this icon in #1355, however this was not merged due to the icon's details not being added to the `appfilter.xml` file (https://github.com/LawnchairLauncher/lawnicons/pull/1356#issuecomment-1647553504).

I have added the icon to the `appfilter.xml` file and changed the file format from an Inkscape SVG to plain SVG.

## Contributor's checklist

- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/owncloud.svg
Comment (Chefski) [inline svgs/owncloud.svg:1]: Could you change the icon to fit the content area?
image

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/owncloud.svg
Comment (Hamster45105) [reply svgs/owncloud.svg:1]: It should be good to go now. I'm new to using Inkscape, but I managed to scale the width of the object up to 160px. Let me know if any more changes need to be made!

Comment (Chefski) [reply svgs/owncloud.svg:1]: The icon got scaled up but the stroke also changed to 8.11 now. A few other people had a problem with this too. Maybe you need to uncheck scaling the stroke too?

Comment (Hamster45105) [reply svgs/owncloud.svg:1]: Should be fixed now. 

The stroke width is 8.000 px and the icon width is 160.000 px.

Comment (Hamster45105) [reply svgs/owncloud.svg:1]: Should be good to go now.

Stroke width: 8px
Icon width: 160px

**Review by Chefski — APPROVED** [review]
Apologies for the slow approval. Thanks :)

**Pull request merged**